x-pack/plugin/mapper-aggregate-metric/src/test/java/org/elasticsearch/xpack/aggregatemetric/mapper/AggregateDoubleMetricFieldMapperTests.java

```diff
@@ -375,8 +372,7 @@ public void testParseArrayValue() throws Exception {
         assertThat(
             e.getCause().getMessage(),
             containsString(
-                "Field [field] of type [aggregate_metric_double] "
-                    + "does not support indexing multiple values for the same field in the same document"
+                "failed to parse field [field.value_count] of type [integer] in document with id '1'. Preview of field's value: '3'"
```

Comment:
Here we receive a very generic error message. I think it's worth throwing the more informative: `Field [field] of type [aggregate_metric_double]  does not support indexing multiple values for the same field in the same documet`. 

Do you think that's possible?